### ISSUE TITLE
Javers Diff - Converting InitialValueChange with JsonConverter throws error

### ISSUE THREAD

**Issue by maffinre** [javers/javers#1092]
Hello Javers-Team,
I found a bug which affects the JsonConverter.
An error is thrown in the JsonConverter when the JSON includes an InitialValueChange.

Example:
I created a Diff of two objects. One has a new child with initial values. This diff is converted to JSON with:
```java
Javers javers = JaversBuilder.javers().withTerminalChanges(false).build();
Diff diff = javers.compare(fooOldRevision, fooCurrent);
javers.getJsonConverter().toJson(diff);
```

The output-JSON of this is:
``` json
{
  "changes": [
    {
      "changeType": "NewObject",
      "globalId": {
        "entity": "cms.domain.Bar",
        "cdoId": "58e90e85-e7c6-47f1-96b2-e71b7d936132"
      }
    },
    {
      "changeType": "InitialValueChange",
      "globalId": {
        "entity": "cms.domain.Bar",
        "cdoId": "58e90e85-e7c6-47f1-96b2-e71b7d936132"
      },
      "property": "id",
      "propertyChangeType": "PROPERTY_VALUE_CHANGED"
    }
  ]
}
```

If I now take this JSON and put it in the method
```java
Diff diff = javers.getJsonConverter().fromJson(changes, Diff.class);
```
to convert it to a Diff again, an error is thrown: `MALFORMED_CHANGE_TYPE_FIELD: no such Change type - 'InitialChange'`

This error seems to be thrown in the ChangeTypeAdapter.class as the InitialValueChange.class is **not** part of the constructor of this class.

Thank you for your help!

**Comment by bartoszwalacik** [comment]
Hi, thanks for reporting, consider contributing a PR with the fix

**Comment by maffinre** [comment]
I added the init but it was not working after that as well. 
After adding the init, it was possible to convert the JSON into a Diff, but all the values of the initialValueChange were null.
It seems that some kind of initialization is still missing after this. 
Could you please look into this as I cannot find the problem?

**Comment by bartoszwalacik** [comment]
The problem is that there is no de-serialization code in javers for that types: `InitialValueChange` and `TerminalValueChange`. See `org.javers.core.json.typeadapter.change.ValueChangeTypeAdapter`. To fix it you need to contribute a PR to javers

**Comment by bartoszwalacik** [comment]
fixed in 6.1.1
